api/logs/src/main/java/io/opentelemetry/api/events/EventLogger.java

```diff
@@ -0,0 +1,144 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.api.events;
```

Comment:
@jkwatson, @trask what are your thoughts on moving the event stuff to an artifact under `/extensions`? Maybe `:extensions:incubator` or a new `:extensions:event-api`?

@mateuszrzeszutek and I both think its a [good idea](https://github.com/open-telemetry/opentelemetry-java/pull/4961#issuecomment-1316662632)

Reply:
no strong opinion either way.  I'm not opposed to wherever you think is appropriate

Reply:
I think either of the `opentelemetry-api-events` and `:extensions:event-api` options is fine; we just shouldn't keep it in the logs module, we're trying to stabilize logs API/SDK right now.